src/functions/template-tags/organizer.php

```diff
@@ -55,6 +55,15 @@ function tribe_get_organizer_ids( $event_id = null ) {
 				$organizer_ids[] = $event_id;
 			} else {
 				$organizer_ids = tribe_get_event_meta( $event_id, '_EventOrganizerID', false );
+
+				// for some reason we store a blank "0" element in this array.
+				// let's scrub this garbage out
+				if ( is_array( $organizer_ids ) ) {
+					$key = array_search( '0', $organizer_ids );
```

Comment:
`array_filter` requires a callback which feels less awesome to me, but I'm open to it if in general we think it is better